app/api/events.py

```diff
@@ -135,7 +135,7 @@ def validate_timezone(self, data):
                                 self_view='v1.event_social_link',
                                 self_view_kwargs={'id': '<id>'},
                                 related_view='v1.social_link_detail',
```

Comment:
This is a wrong relation, as event should not be connected to detail, but list. Fixing this.